<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Okno</class>
 <widget class="QDialog" name="Okno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Styl czcionki</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontStyle">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Kolor czcionki</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontColor"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumTime">
        <time>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Wybierz zdjecie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Okno</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
